dcm4che-core/src/main/java/org/dcm4che3/data/BulkData.java

```diff
@@ -227,4 +228,63 @@ public static Value deserializeFrom(ObjectInputStream ois)
             StringUtils.maskEmpty(ois.readUTF(), null),
             ois.readBoolean());
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
I wanted some comparison capability to make sure even after update the values compare correctly to originally created values.